sc/src/main/webapp/resources/css/atlas.css

```diff
@@ -6,6 +6,10 @@
     font-size: 3em;
 }
 
+#latest-experiments .media-object-section:first-child {
```

Comment:
A slight annoyance is that the padding at the top and bottom of the latest experiments box are not the same. It makes the list look off-center. It would be nice to disable the padding on the last element in the list to fix this, or make the padding at the top match the one on the bottom and left.